pywemo/subscribe.py

```diff
@@ -351,24 +351,33 @@ def _url_resubscribe(
         sid: Optional[str],  # pylint: disable=unsubscriptable-object
         url_fn: SubscribeUrlFn,
     ) -> None:
-        request_headers = headers.copy()
         url = url_fn(device)
+        if sid:
+            # The device will trigger a notification for each new subscription.
+            # A subscription renewal will not trigger this notification. The
+            # notification confirms that the device is subscribed and can
+            # connect to the RequestHandler. Clients can therefore use the
+            # is_subscribed method to confirm that the subscription is alive
+            # and working properly.
+            try:
+                requests.request(
+                    method='UNSUBSCRIBE',
+                    url=url,
+                    headers={'SID': sid},
+                    timeout=REQUESTS_TIMEOUT,
+                )
+            except requests.RequestException:
+                pass
         response = requests.request(
```

Comment:
Is there a possible window between unsubscribing and resubscribing - where we may miss an update?

Reply:
I _think_ this is okay and we won't miss an update. I added some [details below](https://github.com/pavoni/pywemo/pull/236#issuecomment-774325545)